### PR TITLE
Editorial notes from applicability read-through

### PR DESCRIPTION
**Pull request by MikeBishop**
These changes should be largely editorial; I'm happy to pull out anything controversial into a separate PR if desired.

### CODE REVIEW

File: draft-ietf-quic-applicability.md [hunk 1]
```diff
@@ -95,33 +99,32 @@ to QUIC, and implementors of these application protocols.
...
-number of advanced features. While initially designed for the HTTP use case, it
-provides capabilities that can be used with a much wider variety of
-applications. QUIC is encapsulated in UDP. QUIC version 1 integrate TLS 1.3
-{{!TLS13=RFC8446}} to encrypt all payload data and most control information.
-HTTP operating over QUIC is known as HTTP/3.
+QUIC {{QUIC}} is a new transport protocol providing a number of advanced
+features. While initially designed for the HTTP use case, it provides
+capabilities that can be used with a much wider variety of applications. QUIC is
+encapsulated in UDP. QUIC version 1 integrate TLS 1.3 {{!TLS13=RFC8446}} to
+encrypt all payload data and most control information. The version of HTTP which
```
Comment (martinthomson) [inline draft-ietf-quic-applicability.md:106]:
```suggestion
encrypt all payload data and most control information. The version of HTTP that
```

**Review by martinthomson — APPROVED** [review]

File: draft-ietf-quic-applicability.md [hunk 2]
```diff
@@ -95,33 +99,32 @@ to QUIC, and implementors of these application protocols.
...
-QUIC {{!QUIC=I-D.ietf-quic-transport}} is a new transport protocol providing a
-number of advanced features. While initially designed for the HTTP use case, it
-provides capabilities that can be used with a much wider variety of
-applications. QUIC is encapsulated in UDP. QUIC version 1 integrate TLS 1.3
-{{!TLS13=RFC8446}} to encrypt all payload data and most control information.
-HTTP operating over QUIC is known as HTTP/3.
+QUIC {{QUIC}} is a new transport protocol providing a number of advanced
+features. While initially designed for the HTTP use case, it provides
+capabilities that can be used with a much wider variety of applications. QUIC is
+encapsulated in UDP. QUIC version 1 integrate TLS 1.3 {{!TLS13=RFC8446}} to
```
Comment (ianswett) [inline draft-ietf-quic-applicability.md:105]:
```suggestion
encapsulated in UDP. QUIC version 1 integrates TLS 1.3 {{!TLS13=RFC8446}} to
```

File: draft-ietf-quic-applicability.md [hunk 3]
```diff
@@ -95,33 +99,32 @@ to QUIC, and implementors of these application protocols.
...
 and issues that application developers must consider when using QUIC as a
 transport for their application.
 
 # The Necessity of Fallback {#fallback}
 
-QUIC uses UDP as a substrate. This enables both userspace implementation
-traversal of middleboxes and NAT without requiring updates.
+QUIC uses UDP as a substrate. This enables userspace implementation and permits
+traversal of network middleboxes (including NAT) without requiring updates to
+existing infrastructure.
```
Comment (ianswett) [inline draft-ietf-quic-applicability.md:121]:
```suggestion
existing network infrastructure.
```

File: draft-ietf-quic-applicability.md [hunk 4]
```diff
@@ -230,14 +233,14 @@ translation could still affect the path. In particular, firewalls will often
...
 packets from the client.
 
-A QUIC application can adjust idle periods to manage the risk of timeout
-(noting that idle periods and the network idle timeout is distinct from the
-connection idle timeout, defined as the minimum of the idle timeout parameter
-in Section 10.1 of {{QUIC}}), but then there are three options:
+A QUIC application can adjust idle periods to manage the risk of timeout (noting
+that idle periods and the network idle timeout is distinct from the connection
+idle timeout, defined as the minimum of either endpoint's idle timeout
+parameter; see {{Section 10.1 of QUIC}}), but then there are three options:
```
Comment (ianswett) [inline draft-ietf-quic-applicability.md:239]: This sentence is long, can we break it up?

```suggestion
A QUIC applications can adjust idle periods to manage the risk of timeout.
Idle periods and the network idle timeout are distinct from the connection
idle timeout, which is defined as the minimum of either endpoint's idle timeout
parameter; see {{Section 10.1 of QUIC}}). There are three options:
```

File: draft-ietf-quic-applicability.md [hunk 5]
```diff
@@ -317,16 +319,18 @@ same stream. If there is a logical grouping of data chunks or
...
-on a message basis. If a QUIC receiver has maximum allowed concurrent streams
-open and the sender on the other end indicates that more streams are needed, it
-doesn't automatically lead to an increase of the maximum number of streams by
-the receiver. Therefore it can be valuable to expose maximum number of allowed,
-currently open and currently used streams to the application to make the mapping
-of data to streams dependent on this information.
+on a message basis.
+
+If a QUIC receiver has the maximum number of streams open and the sender on the
+other end indicates that more streams are needed, it doesn't automatically lead
```
Comment (ianswett) [inline draft-ietf-quic-applicability.md:325]: In other spots, you mention the frame, so it seems sensible to do so here?

```suggestion
other end indicates that more streams are needed via a STREAMS_BLOCKED frame, it doesn't automatically lead
```

**Review by ianswett — APPROVED** [review]
A few additional suggestions, but these changes LG.

File: draft-ietf-quic-applicability.md (hunk 4 above)
Comment (MikeBishop) [reply draft-ietf-quic-applicability.md:239]: "A QUIC applications" => "QUIC applications" or "A QUIC application"

**Pull request merged**